sources/centos-http.go

```diff
@@ -268,6 +268,11 @@ gpg_keys_official="file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7-aarch64 file:///
 		echo gpgcheck=0 >> /etc/yum.repos.d/cdrom.repo
 	fi
 
+	# Disable fastestmirror plugin
+	if [ -f /etc/yum/pluginconf.d/fastestmirror.conf ]; then
```

Comment:
Why does need to be disabled?

Reply:
I'm not sure whether fastest mirror overrides the set baseurl. I disabled it to make sure it doesn't.